tiledb/sm/array/array.cc

```diff
@@ -128,15 +125,19 @@ Status Array::open_without_fragments(
     EncryptionType encryption_type,
     const void* encryption_key,
     uint32_t key_length) {
-  std::unique_lock<std::mutex> lck(mtx_);
-
+  // Checks
   if (is_open_)
     return LOG_STATUS(Status_ArrayError(
         "Cannot open array without fragments; Array already open"));
-
   if (remote_ && encryption_type != EncryptionType::NO_ENCRYPTION)
+    return LOG_STATUS(
+        Status_ArrayError("Cannot open array without fragments; encrypted "
+                          "remote arrays are not supported."));
+  bool is_array = false;
```

Comment:
This also needs the `!remote_` treatment, I missed tagging this in the other review.